tests/sparseml/exporters/transforms/test_fold_identity_initializers.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import onnx
+
+from sparseml.exporters.transforms import FoldIdentityInitializers
+
+
+def test_fold_identity_initializers():
+    model_input = onnx.helper.make_tensor_value_info(
+        "input", onnx.TensorProto.FLOAT, (1,)
+    )
+    model_output = onnx.helper.make_tensor_value_info(
+        "output", onnx.TensorProto.FLOAT, (1,)
+    )
+
+    init1 = onnx.helper.make_tensor(
```

Comment:
let's maybe move graph construction out into a helper function with a comment on the graph structure